src/Domain/LeanCode.CQRS.Execution/ResponseLoggerElement.cs

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Remember to fix the warnings in this file and in the `DevResponseLoggerElement`.